rdkit/Chem/Descriptors.py

```diff
@@ -162,6 +162,29 @@ def MinAbsPartialCharge(mol,force=False):
 
 _setupDescriptors(locals())
 
+class PropertyFunctor(rdMolDescriptors.PythonPropertyFunctor):
```

Comment:
This python class really isn't necessary is it?
Client code can create functors by subclassing the C++ class directly.